Yoast/YoastSEO.js
# Add Italian linking suggestions

## Summary

This PR can be summarized in the following changelog entry: Adds prominent words for Italian

## Test instructions

This PR can be tested by following these steps:

* Run `grunt build:js`.
* Use the prominent word example in your browser.
* Set language to it_IT.
* Paste a Italian text into the example.
* Make sure you see no strange word combinations, counts, etc.


Fixes #1171
